ts/Extensions/Annotations/MockPoint.ts

```diff
@@ -224,11 +225,12 @@ class MockPoint {
     public static pointToOptions(
         point: Highcharts.AnnotationPointType
     ): Highcharts.AnnotationMockPointOptionsObject {
+
         return {
             x: point.x as any,
             y: point.y as any,
-            xAxis: point.series.xAxis,
-            yAxis: point.series.yAxis
+            xAxis: point.series.xAxis as AxisType,
+            yAxis: point.series.yAxis as AxisType
```

Comment:
I don't think we should do the cast here... We don't _know_ that the series has an axis. What happens if the series is a pie series? Instead of casting, we should have a type check or allow the `AnnottationMockPointOptionsObject.xAxis` and `yAxis` to be undefined.